spikingjelly/activation_based/op_counter/spikesim/config.py

```
@@ -0,0 +1,128 @@
+        r"""
+        **API Language:**
+        :ref:`中文 <SpikeSimEnergyConfig.validate-cn>` |
+        :ref:`English <SpikeSimEnergyConfig.validate-en>`
+
+        ----
+
+        .. _SpikeSimEnergyConfig.validate-cn:
+
+        * **中文**
+
+        校验配置是否合法；不合法时抛出 ``ValueError``。
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF002)